src/open_inwoner/scss/components/Container/Container.scss

```diff
@@ -14,7 +14,9 @@
   }
 
   @media (min-width: 768px) {
```

Comment:
Are we sure about these magic numbers as breakpoints? I see `@media (min-width: 768px) {` here but in the file above there is `@media (min-width: 767px) {`

Reply:
Ah: **_min_**-width should be 768px everywhere, but **_max_**-width should be 767px